frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.scss

```diff
@@ -0,0 +1,16 @@
+.odc-pipeline-task-status {
```

Comment:
After looking at the layout and how the tooltip contents weren't centered, i think this can be changed to use a grid-layout:

```
.odc-pipeline-task-status {
  display: inline-grid;
  grid-gap: var(--pf-global--spacer--sm);
  text-align: left;
  grid-template-columns: 1rem auto;

  &__legend {
    height: 1rem;
    width: 1rem;
  }
}
```